### ISSUE TITLE
Addition of Python Programming Lanuguage in Language Specifics

### ISSUE THREAD

**Issue by abhi98khandelwal** [lnishan/awesome-competitive-programming#41]
As Python 2 and Python 3 have been officially added in the World Finals of ACM ICPC https://icpc.baylor.edu/worldfinals/programming-environment , I would recommend you to add it as well in Language Specifics.

**Comment by lnishan** [comment]
Yes, I'm aware of this, but I'm not quite sure as to what to include in the Python section.

On a quick Google Search shows:

- https://www.quora.com/What-are-some-Python-modules-I-should-know-for-competitive-programming
- https://www.quora.com/What-do-you-think-about-the-induction-of-Python-in-ACM-ICPC-2017

Other links don't seem to be better than an ordinary Python introductory book.

**Comment by lnishan** [comment]
With that said,
I do think including Python is necessary :)

**Comment by abhi98khandelwal** [comment]
what about this http://legacy.python.org/doc/essays/

**Comment by lnishan** [comment]
Well there are certainly plenty of learning resources you can find on the Internet.

However, for this list I want to target at knowledge more specific to competitive programming.
